### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by tewchen**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ Microsoft Defender for Endpoint Device Control Removable Storage Access Control
 |User-based Support     |   Yes      |
 |Machine-based Support    |    Yes     |
 
+## Licensing
+
+Before you get started with Removable Storage Access Control, you should [confirm your Microsoft 365 subscription](https://www.microsoft.com/microsoft-365/compare-microsoft-365-enterprise-plans?rtc=1). To access and use Removable Storage Access Control, you must have the following:
+
+- Microsoft 365 E3 for functionality/policy deployment
+- Microsoft 365 E5 for reporting
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
- Microsoft 365 E3 for functionality/policy deployment.
- Microsoft 365 E5 for reporting.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tewchen, kindly commit the minor suggested changes. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @tewchen @yogkumgit @JohanFreelancer9

**Pull request merged**